sdk/servicebus/service-bus/src/receivers/receiver.ts

```diff
@@ -158,7 +152,7 @@ export class ReceiverImpl<ReceivedMessageT extends ReceivedMessage | ReceivedMes
   }
 
   private _throwIfAlreadyReceiving(): void {
```

Comment:
`_throwIfAlreadyReceiving()` called in `receiveMessages()` and in the `_registerMessageHandler()` before initiating the respective operations.